scripts/cmake/WindowsConfiguration.cmake

```diff
@@ -0,0 +1,36 @@
+# Module to setup Windows specific flags
+
+if (WIN32)
+  # Prebuilt LLVM for Windows doesn't come with CMake files
+  message(STATUS "Detected MS Windows")
+  # Note: This should check for the CMake version instead of OS
+  cmake_policy(SET CMP0079 NEW)
+
+  # std::min fails on windows after including windows.h
+  # this is a workaround
+  add_compile_definitions(NOMINMAX=1)
+
+  find_package (Python COMPONENTS Interpreter)
+  if(NOT Python_FOUND)
+	message(WARNING "Python not found, cmake will assume that it is on path")
+	set(Python_EXECUTABLE python)
+  endif()
+  message(STATUS "Found Python: ${Python_EXECUTABLE}")
+  set(LIBGOMP_LIB "-lgomp -ldl")
+  set(OS_FLEX_SMTLIB_FLAGS "--wincompat")
+  set(OS_X86_INCLUDE_FOLDER "C:/")
+  set(OS_C2GOTO_FLAGS "-D_MSVC")
+
+  if (CMAKE_CXX_COMPILER_ID STREQUAL "Clang")
+	  # There are a LOT of warnings from clang headers
+	  set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -Wno-everything")
+	  set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wno-everything")
```

Comment:
Can we be more precise about what warning we are disabling here?

Reply:
Since my focus was with MSVC I didn't looked very much at the clang compiler. However, it was from various domains and it stopped the compilation process

Reply:
Right, I get it now. Does our regression builds ESBMC with MSVC?

Reply:
Yes it does